src/app/state/account/saga.ts

```diff
@@ -90,6 +90,13 @@ export function* refreshAccountOnParaTimeTransaction() {
   while (true) {
     const { payload } = yield* take(transactionActions.paraTimeTransactionSent)
 
+    // Increase the chance to get updated transactions from API.
+    // Note: We can't rely on nonce to detect pending paratime transactions
+    yield* delay(3000)
+    yield* call(refreshAccount, payload)
+    yield* delay(6000)
+    yield* call(refreshAccount, payload)
+    yield* delay(6000)
```

Comment:
Why we need all these delays? 6000 is to make sure Consensus and ParaTime blocks are finished?

Reply:
Yes, and hopefully nexus indexes them quite fast.
During deposit flow: we receive allowance tx after the first delay already, but usually not deposit tx yet